kernelci.org/content/en/blog/posts/2021/hackfests/index.md

```diff
@@ -0,0 +1,136 @@
+list](https://groups.io/g/kernelci/topics) as well as [LKML](https://lkml.org/)
+and [Twitter](https://twitter.com/kernelci).  Stay tuned!
+
+
+## Connecting the dots
+
+There is a large ecosystem around the Linux kernel which includes testing in
+many shapes and forms: kernel developers, test developers, test system
+developers, OEMs testing fully integrated products...  All these teams of
+people don't necessarily interact with each other very much outside of their
+organisations, and kernel developers aren't necessarily in the habit of writing
+tests as part of their daily work.
+
+Events such as the KernelCI hackfests give a chance for people from these
+different areas to work together on solving common issues and keep the
+ecosystem healthy.  It also helps with shifting the upstream Linux kernel
+development culture towards a more test-driven workflow, to bring mainline
+Linux closer to the real world where it is actually being used.
+
+Let's consider a plausible hackfest story.  A first participant writes a new
+test, say in kselftest.  A second participant enables the test to run in
+KernelCI, which fails in some cases and a kernel bug is found.  A third
+participant makes a fix for the bug, which can then be tested directly in
+KernelCI to confirm it works as expected.  This may even happen on the [staging
+instance](/docs/instances/staging/) before the patches for the test and the fix
+are sent to any mailing list, in which case the fix would get a `Tested-by:
+"kernelci.org bot" <bot@kernelci.org>` trailer from the start.  This scenario
+also relies on some hardware previously made available in test labs by other
+people, putting together efforts from at least four different participants.
+
+
+## Timeline
+
+Here's a summary of the first two hackfests:
+
+### Hackfest #1 - 27th May to 4th June 2021
+
+* Workboard: https://github.com/orgs/kernelci/projects/3
+* Participans
+  * Several kernel developers from [Google Chrome
+  OS](https://www.google.com/intl/en_uk/chromebook/chrome-os/)
+  * Several kernel developers from [Collabora](https://collabora.com)
+  * A few members of the core KernelCI team
+* Achievements
+  * New KernelCI instance for Chrome OS on https://chromeos.kernelci.org
+  * Added support for building Chrome OS configs on top of mainline
+  * Enabled clang-13 builds for Chrome OS and main KernelCI builds
+  * New test suite for libcamera enabled on https://linux.kernelci.org
+  * Enabled LTP crypto tests with extra kernel config fragment
+  * Several patches were also sent to extend kselftest and KUnit coverage in
+      the kernel tree
+
+### Hackfest #2 - 6th to 10th September 2021
+
+* Workboard: https://github.com/orgs/kernelci/projects/5
+* Participants
+  * More members of the [TSC](/docs/org/tsc/) took part than first hackfest
+  * New contributor: Denis Efremov (floppy disk kernel maintainer)
+  * Alice from [Cybertrust Japan](https://www.cybertrust.co.jp/english/)
```

Comment:
@gctucker Sorry, please add both CIP and Cybertrust Japan also for me. Similar to Chris profile